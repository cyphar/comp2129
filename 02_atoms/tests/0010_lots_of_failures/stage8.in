LOAD what
LOAD
LOAD aa a aa aa
LOAD a.a
START what
HELP
START 2 1 2
START no
START 2 1 a
LOAD shouldnotwork
LOAD
LOAD  a a aaa
QUIT
