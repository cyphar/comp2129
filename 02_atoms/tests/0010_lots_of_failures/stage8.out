Cannot Load Save

Missing Argument

Too Many Arguments

Game Loaded

Invalid Command

Invalid Command

Invalid Command

Invalid Command

Invalid Command

Invalid Command

Invalid Command

Invalid Command

Bye!
